model/src/main/resources/reference.conf

```diff
@@ -117,6 +117,7 @@ ort {
 
         deltaScans = "true"
         deltaScanLimit = "10"
+        waitRepetition = "360"
```

Comment:
I am confused what this value actually means and how it is related to the maximum amount of time we wait for a scan. For usability, wouldn't it be better to specify the timeout directly? The property would then have to be named accordingly.

Reply:
This is exactly my new idea. I will change the implementation.